plugins/modules/organization.py

```diff
@@ -87,11 +171,50 @@ def main():
             name=dict(required=True),
             description=dict(),
             label=dict(),
+            upstream_type=dict(required=False, choices=['redhat_cdn', 'export_sync', 'network_sync']),
+            upstream_url=dict(required=False),
+            upstream_username=dict(required=False),
+            upstream_password=dict(required=False,no_log=True),
+            #upstream_ca_cert=dict(required=False, type='entity', resource_type='content_credentials', scope=['']),
```

Comment:
The Lint job complains. This should fix it.

```suggestion
            upstream_password=dict(required=False, no_log=True),
            # upstream_ca_cert=dict(required=False, type='entity', resource_type='content_credentials', scope=['']),
```